src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -797,6 +813,33 @@ export const GlobalProvider = ({
         [modifyNode]
     );
 
+    useHotkeys(
+        'ctrl+c, cmd+c',
+        () => {
+            const selectedNodes = getNodes().filter((n) => n.selected);
+            const selectedEdges = getEdges().filter((e) => e.selected);
+            const copyData = JSON.stringify({ nodes: selectedNodes, edges: selectedEdges });
+            clipboard.writeText(copyData, 'clipboard');
```

Comment:
This will allow users to just paste nodes as text. Worse, chaiNNer will interpret any text currently in the clipboard to be JSON it can parse. That's a massive usability issue.

As the MIME type, use something like `application/chainner.chain`.

Reply:
The only way to do that is to change this to writeBuffer